backend/db/refresh_dataset_group_tables.py

```diff
@@ -64,14 +64,20 @@ def refresh_voc_and_counts(skip_downloads: bool = False, schema=SCHEMA):
             for statement in statements:
                 run_sql(con, statement)
             # print('Recreating derived tables')  # printed w/in refresh_derived_tables()
-            refresh_any_dependent_tables(con, config['tables'])
-
+            refresh_derived_tables(con, config['tables'], schema)
+            print('Deleting old, temporarily backed up versions of tables')
```

Comment:
## Bugfix: "Can't drop tables because dependencies"
Needed to move this to after remaking derived tables. No idea how this script was able to succeed before without this step being at the end.